recipe/bld.bat

```diff
@@ -22,6 +22,7 @@ cmake %CMAKE_ARGS% ^
       -DJPEGXL_STATIC:BOOL=OFF ^
       -DJPEGXL_FORCE_SYSTEM_BROTLI:BOOL=ON ^
       -DJPEGXL_FORCE_SYSTEM_HWY:BOOL=ON ^
+      -DJPEGXL_FORCE_SYSTEM_LCMS2:BOOL=ON ^
```

Comment:
Should we bring back the discussion on `skcms` vs. `lcms2`? The official doc seems to [prefer `skcms`](https://github.com/libjxl/libjxl/blob/main/doc/color_management.md#cms-library). If we are switching to `lcms2` here, we should also remove the `JPEGXL_ENABLE_SKCMS` option and the source in `meta.yaml`.

Reply:
no sorry we should not. this wasn't intended to. I was trying to address a potentially out of date note that `lcms2` was not available on all platforms at some time. Let me remove lcms2